### PR TITLE
update instructions to latest

### PR DESCRIPTION
**Pull request by dholbach**
fixes: #496

### CODE REVIEW

**Comment by chanwit** [comment]
thank you for this PR @dholbach 
a few change to reflect the new version of Firecracker :-)

**Comment by dholbach** [comment]
Might be worth automating this somehow? Maybe a `sed` in a release script somewhere? :-)

File: docs/installation.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ If the installation was successful, the `ignite` command should now be available
 
 ```console
 $ ignite version
-Ignite version: version.Info{Major:"0", Minor:"6", GitVersion:"v0.6.2", GitCommit:"...", GitTreeState:"clean", BuildDate:"...", GoVersion:"...", Compiler:"gc", Platform:"linux/amd64"}
+Ignite version: version.Info{Major:"0", Minor:"6", GitVersion:"v0.6.3", GitCommit:"...", GitTreeState:"clean", BuildDate:"...", GoVersion:"...", Compiler:"gc", Platform:"linux/amd64"}
 Firecracker version: v0.18.0
```
[the remark this answers is not in the document]
Comment (dholbach) [reply docs/installation.md]: You're right! Confirmed!

**Comment by chanwit** [comment]
LGTM

**Comment by chanwit** [comment]
Thank you @dholbach :-) :-) :-)

**Comment by stealthybox** [comment]
AAAAAHHH I'm always forgetting to bump this number before I release >_>

We need add this to the doc generation

**Pull request merged**